cmd/kube-proxy/app/server_windows.go

```diff
@@ -101,75 +96,51 @@ func newProxyServer(config *proxyconfigapi.KubeProxyConfiguration, master string
 		healthzPort, _ = strconv.Atoi(port)
 	}
 
+	// Check if Kernel Space can be used.
+	canUseWinKernelProxy, err := winkernel.CanUseWinKernelProxier(winkernel.WindowsKernelCompatTester{})
```

Comment:
For documentation purposes, it might be nice to comment here why this might fail.  @jayunit100 do we know why it might?

Reply:
cc @jsturtevant do you have more context of cases where winkernel is not supported?
I can open a PR in the docs as a follow up.